generators/php/sdk/src/endpoint/http/HttpEndpointGenerator.ts

```diff
@@ -94,6 +108,16 @@ export class HttpEndpointGenerator extends AbstractEndpointGenerator {
         });
     }
 
+    private getClassReference(requestBody: HttpRequestBody): php.ClassReference {
+        return requestBody._visit({
+            inlinedRequestBody: () => this.context.getJsonApiRequestClassReference(),
+            reference: () => this.context.getJsonApiRequestClassReference(),
+            fileUpload: () => this.context.getMultipartApiRequestClassReference(),
+            bytes: () => this.context.getJsonApiRequestClassReference(),
```

Comment:
Let's leave a TODO here - we'll need to roll out a separate request type (e.g. `BytesApiRequest`) to support the `bytes` type, but we can handle that in a follow-up.
```suggestion
            bytes: () => this.context.getJsonApiRequestClassReference(), // TODO: Add support for BytesApiRequest.
```

Reply:
done